.babelrc

```diff
@@ -8,6 +8,8 @@
     },
     "test": {
       "presets": [
+        "react",
+        "es2015",
```

Comment:
No need of these because they are already present in `next/babel` preset we added below